src/Cake.Core/Scripting/Processors/UriDirectiveProcessor.cs

```diff
@@ -6,12 +6,15 @@
 using System.Collections.Generic;
 using System.Linq;
 using System.Text;
+using System.Text.RegularExpressions;
 using Cake.Core.Scripting.Analysis;
 
 namespace Cake.Core.Scripting.Processors
 {
     internal abstract class UriDirectiveProcessor : LineProcessor
     {
+        private Regex uriPrefixPattern = new Regex("^([a-zA-Z]{2,}:)");
```

Comment:
Could you move the assignment to the constructor of this one and rename it to `_uriPrefixPattern`. The variable should also be `readonly`. Thanks.

Reply:
Done